protocol/xmrmaker/swap_state.go

```diff
@@ -109,6 +109,14 @@ func newSwapState(b backend.Backend, offer *types.Offer, om *offerManager, statu
 	return s, nil
 }
 
+func (s *swapState) lockState() {
+	s.stateMu.Lock()
+}
+
+func (s *swapState) unlockState() {
+	s.stateMu.Unlock()
+}
```

Comment:
If there is are alternative names that you like better, just let me know and I'll rename these.